spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/configuration/JobDependencies.java

```diff
@@ -211,16 +210,16 @@ public SimpleJobServiceFactoryBean simpleJobServiceFactoryBean(DataSource dataSo
 
 	@Bean
 	public JobRepositoryFactoryBean jobRepositoryFactoryBeanForServer(DataSource dataSource,
-			DataSourceTransactionManager dataSourceTransactionManager) {
+			PlatformTransactionManager platformTransactionManager) {
 		JobRepositoryFactoryBean repositoryFactoryBean = new JobRepositoryFactoryBean();
 		repositoryFactoryBean.setDataSource(dataSource);
-		repositoryFactoryBean.setTransactionManager(dataSourceTransactionManager);
+		repositoryFactoryBean.setTransactionManager(platformTransactionManager);
 		return repositoryFactoryBean;
 	}
 
 	@Bean
-	public DataSourceTransactionManager transactionManagerForServer(DataSource dataSource) {
-		return new DataSourceTransactionManager(dataSource);
+	public PlatformTransactionManager transactionManager(EntityManagerFactory entityManagerFactory) {
```

Comment:
This is probably not needed anymore as it should come from boot. At latest it's going to get removed when we go to boot21. We added custom when we were on boot14 and boot15 added manager config.

Reply:
Yeah, probably once we update to Boot 2.1.x. Currently, it is needed to make the transaction active inside the tests.